src/remote/utils.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const Qs = require('qs')
+const mapValues = require('lodash.mapvalues')
+
+exports.getResponse = (res, cb) => {
+  let data = ''
+  res.on('data', function (buf) {
+    data += buf.toString()
```

Comment:
Since one character can be represented by more than one byte, you should just concatenate buffers, not strings.